app/controllers/imports_controller.rb

```diff
@@ -5,7 +5,9 @@ class ImportsController < ApplicationController
   # GET /imports
   def index
     @imports = @podcast.podcast_imports.order(created_at: :asc)
-    @import = @podcast.podcast_imports.new
+    @import = @podcast.podcast_imports.new(import_params)
+    @import.clear_attribute_changes(%i[type])
```

Comment:
is `clear_attribute_changes` just a way to make sure that `type` stays what it was input as from `import_params`? does this have to do with the `reverse_merge` default that occurs below?

Reply:
It's cosmetic in this case - I was trying to get the field to not have an orange border when the page first loads.